examples/strands/strands.bal

```diff
@@ -1,20 +1,26 @@
 import ballerina/io;
 
+function userSpeakerService(string userName) {
+    string messagePrefix = userName + " speaking : ";
+    foreach int i in 0...9 {
+       io:println(messagePrefix + i.toBalString());
```

Comment:
`io:println(messagePrefix, i);`
Isn't this better?

Reply:
yes indeed. Thanks

Reply:
Added by commit b5d89d22fe031f2ee28f05e584454e8b6f38c441